tests/helpers/testhelper.js

```diff
@@ -126,7 +132,8 @@ exports.setup.testRun = function (test, name) {
 
       test.ok(
         undefinedErrors.length === 0
-          && unthrownErrors.length === 0 && wrongLineNumbers.length === 0,
+          && unthrownErrors.length === 0 && wrongLineNumbers.length === 0
+          && duplicateErrors.length === 0,
```

Comment:
A quick ctrl-f of this file shows that this is the only place where the logical operator is at the beginning of the line.